service/src/main/java/uk/gov/hmcts/reform/fpl/model/SupportingEvidenceBundle.java

```diff
@@ -17,7 +17,7 @@
 public class SupportingEvidenceBundle {
     private final String name;
     private final String notes;
-    @PastOrPresent(message = "Date of time received cannot be in the future")
+    @DateNotInFuture()
```

Comment:
@PastOrPresent is used in 3 other places, should we change it too? (probably a different tech debt task!)

Reply:
No because the other places it validates LocalDate rather than LocalDateTime which works fine.

Reply:
@KiranVarma02 problem is when PastOrPresent is on LocalDateTime, if it is on LocalDate as in other 3 places it is a bit different problem (less likely)